### ISSUE TITLE
Follow links broken in FF 51.0.1

### ISSUE THREAD

**Issue by jonross** [vimperator/vimperator-labs#678]
##### Issue type:

- Bug report

##### Version:

```
Vimperator: 3.15.0 (created: 2016/10/05 01:11:26)
FIrefox:    Mozilla/5.0 (Macintosh; Intel Mac OS X 10.0; rv:51.0) Gecko/20100101 Firefox/51.0
```

##### Description:

'f' command no longer works.  It prompts for a link but nothing is highlighted instead there is an error:
"Reference error: tabs is undefined"

##### Expected behavior:

Normal documented behavior

**Comment by timss** [comment]
Thank you submitting a bug report, but this is a duplicate of #568 and fixed in #675

**Comment by jonross** [comment]
Thanks, I've successfully gotten around this by running the Nightly release of Firefox + building my own .xpi
